app/tasks/install.php

```diff
@@ -42,11 +43,14 @@
         $defaultHTTPSPort = '443';
         $vars = [];
 
-        /**
-         * We are using a random value every execution for identification.
-         * This allows us to collect information without invading the privacy of our users.
-         */
-        $analytics = new GoogleAnalytics('UA-26264668-9', uniqid('server.', true));
+        $analyticsOption = App::getEnv('_APP_OPTIONS_ANALYTICS', 'enabled');
+        if($analyticsOption !== 'disabled') {
+            /**
+             * We are using a random value every execution for identification.
+             * This allows us to collect information without invading the privacy of our users.
+             */
+            $analytics = new GoogleAnalytics('UA-26264668-9', uniqid('server.', true));
```

Comment:
`$analytics` has an [`enabled` flag](https://github.com/utopia-php/analytics/blob/7f4f7688090d1e4e05af95c693f9a8f2d898c87f/src/Analytics/Adapter/GoogleAnalytics.php#L76) that can be used to disable sending events. That is probably a better approach than adding disabled. checks for every `$analytics->createEvent()` call.

Reply:
To be honest, I'm quite not sure what the install task is for and therefore am not sure how to test it properly. Just adding `$analytics->enabled = false;` after creation of the `$analytics` object currently throws an error as the `enabled` property is protected.

Other than that, I am too not really happy with my implementation and would love to change that to a simple boolean solution. You got any pointers for me?

Reply:
Just tested `$analytics->disabled()` and it works as expected. I've already pushed a new version to the PR. Thanks @stnguyen90!